qemu/tests/qemu_guest_agent.py

```diff
@@ -126,11 +143,62 @@ def _check_ga_service(self, session, cmd_check_status):
         :param session: use for sending cmd
         :param cmd_check_status: cmd to check if ga service is started
         '''
-        error_context.context("Check whether qemu-ga service is started.", logging.info)
+        error_context.context("Check whether qemu-ga service is started.",
+                              logging.info)
         s, o = session.cmd_status_output(cmd_check_status)
         return s == 0
 
     @error_context.context_aware
+    def _get_latest_pkg(self):
+        """
+        get latest qemu-guest-agent rpm package url.
+        :return: rpm pkg list
+        """
+        virt_module_stream = self.params.get("virt_module_stream", "")
+        guest_name = self.params.get("guest_name")
+        arch = self.params["vm_arch_name"]
+        download_root = self.params["download_root_url"]
+        query_timeout = 180
+
+        error_context.context("Check if brew command is presented.",
```

Comment:
better to use `avocado.utils.path.find_command`

Reply:
Would it be better to cancel the test if `brew` is not present?

Reply:
modified.